ehr/src/org/labkey/ehr/EHRModule.java

```diff
@@ -256,6 +256,9 @@ public void moduleStartupComplete(ServletContext servletContext)
         EHRService.get().registerMoreActionsButton(new MarkCompletedButton(this, "study", "parentage", "End Selected Calls", true), "study", "parentage");
         EHRService.get().registerMoreActionsButton(new ExcelImportButton(this, "study", "parentage", "Import Data"), "study", "parentage");
 
+        EHRService.get().registerMoreActionsButton(new MarkCompletedButton(this, "study", "geneticAncestry", "End/Disable Selected", true), "study", "geneticAncestry");
```

Comment:
These should go into ONPRC_EHRModule instead.

Reply:
done